ts/features/payments/bizEventsTransaction/screens/PaymentsTransactionBizEventsDetailsScreen.tsx

```diff
@@ -97,6 +110,14 @@ const PaymentsTransactionBizEventsDetailsScreen = () => {
   };
 
   const handleDownloadPdfReceipt = () => {
+    analytics.trackPaymentsDownloadReceiptAction({
+      organization_name: paymentAnalyticsData?.receiptOrganizationName,
+      first_time_opening: paymentAnalyticsData?.receiptFirstTimeOpening,
+      user:
+        userFiscalCode === paymentAnalyticsData?.receiptPayerFiscalCode
+          ? "payer"
+          : "payee"
```

Comment:
This has been used twice. What about grouping the same code in an utility function? What do you think?

Reply:
Now that you have use `isPayer` is much better. Anyway you still have three lines of code repeated. If you prefer leaving things like they are now, for me it's good. Btw grouping them in a single place'd be better in the long run. What do you think?

Reply:
You're right, what do you think about what implemented in the following commit?
75534e1fb5dd0203628f2a7a955422b15b3ce49b